feat: enterprise pricing admonitions (#468)
loft-sh/vcluster-docs: vcluster/_partials/admonitions/pro-ultimate-admonition.mdx

import Admonition from '@theme/Admonition';

<br></br>
<div class="enterprise-ultimate">
:::info Enterprise-Only Feature

> Description: This feature is available exclusively in vCluster Enterprise Ultimate plans. Learn more about our [pricing plans](https://www.vcluster.com/pricing) or [contact our sales team](https://www.vcluster.com/enterprise-demo) to explore how vCluster Enterprise can support your needs.

Reviewer: I do not think we need to include the word "description". Also this has more of a marketing feel to it I think. I think for the tech doc the following is simpler and to the point:

```suggestion
This feature is only available in the vCluster Enterprise Advanced and Ultimate plans. See our [pricing plans](https://www.vcluster.com/pricing) or [contact our sales team](https://www.vcluster.com/enterprise-demo) for more information.
```

Author: Same here, this should be `This feature is only available in the vCluster Enterprise Ultimate plan.

Reviewer: Apologies, also did a quick copy and paste. Sorry. updated the text.
